### PR TITLE
Change shortcuts

### PR DESCRIPTION
**Pull request by fxsalazar**

### CODE REVIEW

File: src/components/panels.tsx [hunk 1]
```diff
@@ -52,40 +53,55 @@ function Root({ children }: React.PropsWithChildren) {
...
-        panelElement.contains(document.activeElement),
-      )
-
-      const focusedPanelIndex = focusedPanelElement
-        ? panelElements.indexOf(focusedPanelElement)
-        : -1
-
-      if (event.key === "ArrowLeft" && event.altKey) {
-        const firstPanel = panelElements[0]
+    // Focus prev/next panel with `option + [shift] + left/right`
```
Comment (colebemis) [inline src/components/panels.tsx:56]:
```suggestion
    // Focus prev/next panel with `option + shift + left/right`
```

File: src/components/panels.tsx [hunk 2]
```diff
@@ -52,40 +53,55 @@ function Root({ children }: React.PropsWithChildren) {
...
+      Array.from(document.querySelectorAll<HTMLElement>("[data-panel]"))
+    const getPanelsAndFocusedPanelIndex = () => {
+      const panelElements = getPanelElements()
+      return {
+        panelElements,
+        focusedPanelIndex: panelElements.findIndex((panelElement) =>
+          panelElement.contains(document.activeElement),
+        ),
+      }
+    }
```
Comment (colebemis) [inline src/components/panels.tsx:67]: Nitpick: The function abstraction doesn't feel quite right here. What if we made `getFocusedPanelIndex` a function of `panelElements`:

```suggestion
    const getFocusedPanelIndex = (panelElements: Array<HTMLElement>) => {
      return panelElements.findIndex((panelElement) => panelElement.contains(document.activeElement)) 
    }
```

File: src/utils/shortcuts.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  | "next-panel"
+  | "prev-panel"
+  | "first-panel"
+  | "last-panel"
+  | "copy-note-id"
+  | "copy-markdown"
+  | undefined
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getShortcut = (event: any): Shortcut => {
```
Comment (colebemis) [inline src/utils/shortcuts.ts:11]: The naming of this function was confusing to me. In my view, this function accepts a _shortcut_ and returns the associated _action_.

What if we called this function `getKeyboardAction`?
```suggestion
export const getKeyboardAction = (event: any): KeyboardAction => {
```

**Comment by fxsalazar** [comment]
Yes, I'll get back to it this week...

**Review by colebemis — COMMENTED** [review]
Thanks for your patience, @fxsalazar! 

I think `CMD + ENTER` to open a new note dialog conflicts with saving a note. Can we change that to something else? I'm open to ideas.

The other shortcuts look good.

**Comment by zaaakher** [comment]
@colebemis @fxsalazar I'd suggest `CMD + N` for **n**ew **n**ote

**Comment by fxsalazar** [comment]
Hello, so sorry for the delay...

> I think `CMD + ENTER` to open a new note dialog conflicts with saving a note. Can we change that to something else? I'm open to ideas.

Well, I did it and tested it realizing that those kind of shortcuts are bind to the context; and it came naturally, while finishing editing a note and then wanting to create a new note I said that it was in the flow to use the same keys. Else, we could use the `ctrl` key instead 🤷🏽 

@zaaakher cmd+N opens a new browser window 😞

**Comment by colebemis** [comment]
@fxsalazar I'm working on a larger refactor of the keyboard shortcuts and keyboard navigation system so I'm going to close this PR to avoid conflicts. 

Thank you for the work you put into this ❤️  It's been a helpful reference for this larger refactor.

**Pull request closed without merging**